src/frontends/lean/structure_cmd.cpp

```diff
@@ -621,8 +625,10 @@ struct structure_cmd_fn {
                 default_value = m_p.parse_expr();
             }
         }
-        if (default_value && !is_explicit(bi)) {
-            throw parser_error("invalid field, it is not explicit, but it has a default value", start_pos);
+        if (default_value) {
+            if (!is_explicit(bi))
+                throw parser_error("invalid field, it is not explicit, but it has a default value", start_pos);
+            default_value = some(mk_annotation(*g_default_val_body, *default_value));
```

Comment:
@leodemoura So I guess fixing this issue via annotations doesn't work at trust level 0. I'm out of ideas, thoughts?